doc/user/bgp.rst

```diff
@@ -1550,7 +1550,8 @@ is specified, the BGP protocol process belongs to the default VRF.
 
 BGP routes may be leaked (i.e., copied) between a unicast VRF RIB and the VPN
 safi RIB of the default VRF (leaking is also permitted between the unicast RIB
-of the default VRF and VPN).  A common application of this feature is to
+of the default VRF and VPN) or routes may be leaked directly between two BGP
+VRF instances.  A common application of the VPN-VRF feature is to
 connect a customer's private routing domain to a provider's VPN service.
 Leaking is configured from the point of view of an individual VRF: ``import``
 refers to routes leaked from VPN to a unicast VRF, whereas ``export`` refers to
```

Comment:
Rather than this change, I propose the following one-sentence paragraph following this paragraph:

A shortcut syntax is also available for specifying leaking from one vrf to another using the VPN RIB as intermediary.

Reply:
this looks like it's in already.  I do not need to do this, right?

Reply:
actually I read this wrong.  Done.